bob/src/cluster/operations.rs

```diff
@@ -281,8 +331,12 @@ pub(crate) async fn put_local_node(
 pub(crate) async fn delete_at_local_node(
     backend: &Backend,
     key: BobKey,
+    vdisk_id: VDiskId,
+    disk_path: DiskPath,
+    force_delete: bool,
```

Comment:
No need to pass this flag. Local delete is always assumed to be forced (always write deletion record)